Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -12,7 +12,8 @@
     }
     <Form>
 
-        @if ( IsCellEdit && ( CommandColumn is null || !CommandColumn.SaveCommandAllowed ) )
+        @if ( (IsCellEdit && (CommandColumn is null || !CommandColumn.SaveCommandAllowed))

```

Comment:
Can we move this entire if statement under the getter property and name it according to what it does? For future clarity.